explorer/src/components/layout/Layout.tsx

```diff
@@ -1,6 +1,6 @@
 'use client'
 
-import { CookieBanner } from 'components/common/CookieBanner'
+// import { CookieBanner } from 'components/common/CookieBanner'
```

Comment:
Why removing CookieBanner?

Reply:
had added for some testing, removed it now